src/query_part.ts

```diff
@@ -155,84 +142,23 @@ rootAggregations.forEach((aggregation) => {
     params: [],
     defaultParams: [],
   });
-  // TODO: figure out tag arguments
+
   registerGroupBy({
     type: aggregation,
     addStrategy: replaceAggregationAddStrategy,
     categoryName: "Group By",
-    params: [[]],
+    dynamicParameters: true,
```

Comment:
I changed the base queryPartDef we use to include something called `dynamicParameters`. It is a bit hacky right now, but essentially its a flag that tells the `queryPartEditor` that it should duplicate the first parameter definition whenever it is set, so that any number of tags can be set on a group by clause

Reply:
What about calling this something like `preserveTags` or `preserveParameters`?

Reply:
I’m not sure that variable name makes sense. It doesn’t “preserve” anything, it enables dynamically adding new parameters to the action.